Command/ClearCommand.php

```diff
@@ -38,34 +39,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->setObjectManagerFromInput($input);
 
-        $toReindex = [];
+        $toClear = [];
 
-        if ($input->hasArgument('entityName')) {
+        if ($input->getArgument('entityName')) {
             $filter = $this->getObjectManager()->getRepository($input->getArgument('entityName'))->getClassName();
         } else {
             $filter = null;
         }
 
         foreach ($this->getObjectClasses() as $class) {
             if (!$filter || $class === $filter) {
-                $toReindex[] = $class;
+                $toClear[] = $class;
             }
         }
 
-        $nIndexed = 0;
-        foreach ($toReindex as $className) {
-            $nIndexed += $this->clear($className);
+        $nCleared = 0;
+        foreach ($toClear as $className) {
+            try {
+                $this->clear($className);
+                $nCleared++;
```

Comment:
Same here, I'm not sure about this. I'd rather keep the return statement in `clear()` and add the result to `$nCleared`.